tests/scripts/all.sh

```diff
@@ -154,9 +159,12 @@ cleanup()
     fi
 
     command make clean
+    cd crypto
+    command make clean
+    cd ..
```

Comment:
Non-blocker: this can break if `crypto` is a symlink. Replace these three lines by `make -C crypto clean`. Other places that use `cd crypto; …; cd ..` should be similarly protected.

Reply:
Note: An option (for commands other that actually require changing the current working directory) is to use a sub-shell: `(cd crypto; do_stuff; more_stuff)`.